edk2toolext/versioninfo/versioninfo_tool.py

```diff
@@ -0,0 +1,67 @@
+# @file versioninfo_tool.py
+# This module contains the CLI interface for easily creating VERSIONINFO resources
+# from json files.
+#
+##
+# Copyright (C) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+
+import os
+import sys
+import argparse
+import json
+from edk2toolext.versioninfo.versioninfo_helper import PEObject, VERSIONINFOGenerator
+
+TOOL_DESCRIPTION = """
+Versioninfo Tool is a command-line tool to assist in generating VERSIONINFO
+resource files for use with Resource Compiler. It takes a JSON representing
+versioning info and produces a resource file that satifies UEFI SBAT requirements
+and is compatible with Resource Compiler.
+
+An example call might look like:
+%s -o /path/to/version.JSON /path/to/output
```

Comment:
so since this is in extensions, we'll expose a console script. So it'll be binver_encode or binver_decode or something like that